### PR TITLE
Schema erwin

### PR DESCRIPTION
**Pull request by eamador1**
The following changes were made in this branch:
---

- create tables item, music_album and genre in schema.sql

### CODE REVIEW

**Review by Estete9 — CHANGES_REQUESTED** [review]
Hi @eamador1 
I'll review this PR. ✋

Good job so far it looks quite good!
There are some recommendations I would like you to take into account.

Highlights
 You have written a clean code ✅
 The distribution, padding, and margins are perfect. ✅
 Good PR title/description ✅
Required Changes ♻️
 Good job adding the elements in the list in the main section, but we can improve it a little bit more together. 💪 
 
Check the comments under the review.

Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @Estete9 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/Estete9">
   @Estete9
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/esteban-palacios-5030a772/">
   linkedin-esteban-palacios-5030a772
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @NaughTban
</a>

</td>
</tr>

**Comment by eamador1** [comment]
The following changes were made since the last code review:
---

- Delete item table and add inherited properties to music_album table

**Review by Estete9 — CHANGES_REQUESTED** [review]
Hi @eamador1 
I'll review this PR. ✋

Good job so far it looks quite good!
There are some recommendations I would like you to take into account.



Highlights
 You have written a clean code ✅
 The distribution, padding, and margins are perfect. ✅
 Good PR title/description ✅
Required Changes ♻️
 Good job adding the elements in the list in the main section, but we can improve it a little bit more together. 💪 
 
Check the comments under the review.

Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @Estete9 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/Estete9">
   @Estete9
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/esteban-palacios-5030a772/">
   linkedin-esteban-palacios-5030a772
</a>

</td>
<td>

<a href="https://twitter.com/NaughTban">
   @NaughTban
</a>

</td>
</tr>

File: schema.sql [hunk 1]
```diff
@@ -19,3 +19,20 @@ CREATE TABLE authors(
...
+CREATE TABLE music_album (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    genre VARCHAR(25),
+    author VARCHAR(50),
+    label VARCHAR (50),
+    publish_date  DATE,
+    on_spotify BOOLEAN, 
+    archived BOOLEAN,
+    PRIMARY KEY(id)
+);
```
Comment (Estete9) [inline schema.sql:32]: Great job including the properties from the item class in the music_album table.
Please help me by adding some more properties to the table. We need to add the genre_id, author_id, and label_id.
Also please add the foreign keys with the references to the genres(id), authors(id), and labels(id)

Guide yourself with the `games` table implemented by Jose

**Comment by eamador1** [comment]
Since the last code review the following changes were made:
---

- add relation between music_album table and genre, label and author tables ✅

**Review by Estete9 — APPROVED** [review]
## Hi @eamador1  , 

Your feature is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Great use of modules ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @Estete9 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/Estete9">
   @Estete9
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/esteban-palacios-5030a772/">
   linkedin-esteban-palacios-5030a772
</a>

</td>
<td>

<a href="https://twitter.com/NaughTban">
   @NaughTban
</a>

</td>
</tr>
</table>

**Pull request merged**